src/components/calendar/GoogleCalendarButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
I believe this whole file is leftover from when we fixed this before, this needs to be deleted!